heuristic/repair_operators.py

```diff
@@ -469,7 +487,10 @@ def worst_week_regret_repair(shifts_in_week, competencies, t_covered_by_shift,
                                         for t1, v1 in shifts_in_week[j]
                                         if (t1,v1,comp) not in impossible_shifts
                         }
-        shift = max(shifts.items(), key=itemgetter(1))[0]
+
+        best_shift_value = max(shifts.items(), key=itemgetter(1))[1]
+        shift = choice([key for key, value in shifts.items() if value == best_shift_value])   
```

Comment:
This is to randomly select a shift if there are multiple shifts with the same value?

Reply:
Correct